src/effects/native/phasereffect.cpp

```diff
@@ -64,20 +51,32 @@ EffectManifest PhaserEffect::getManifest() {
     stages->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     stages->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     stages->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    stages->setDefault(3.5);
     stages->setMinimum(1.0);
     stages->setMaximum(6.0);
+    stages->setDefault(3.5);
 
-    EffectManifestParameter* frequency = manifest.addParameter();
-    frequency->setId("lfo_frequency");
-    frequency->setName(QObject::tr("Rate"));
-    frequency->setDescription(QObject::tr("Controls the speed of the low frequency oscilator."));
-    frequency->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
-    frequency->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
-    frequency->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    frequency->setDefault(0.5);
-    frequency->setMinimum(0.0);
-    frequency->setMaximum(5.0);
+    EffectManifestParameter* fb = manifest.addParameter();
+    fb->setId("feedback");
+    fb->setName(QObject::tr("Feedback"));
+    fb->setDescription(QObject::tr("Controls how much of the output signal is looped"));
+    fb->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    fb->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    fb->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    fb->setMinimum(-0.95);
+    fb->setMaximum(0.95);
+    fb->setDefault(0.0);
+
+    EffectManifestParameter* depth = manifest.addParameter();
+    depth->setId("depth");
+    depth->setName(QObject::tr("Depth"));
+    depth->setDescription("Controls the intensity of the effect.");
+    depth->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    depth->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    depth->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    depth->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
+    depth->setMinimum(0.5);
```

Comment:
Listening closer, I do hear an effect at lower values, but it is subtle. I changed it because I don't think it's good to have to turn up the metaknob half way to really start noticing an effect. Maybe this parameter should have a logarithmic scale?